src/less/mixins/dropdown/base/dropdown-mixins.less

```diff
@@ -9,7 +9,7 @@
 .dropdown-overlay-constraints() {
     max-height: 400px;
     overflow-y: auto;
-    z-index: 1;
+    z-index: 2;
```

Comment:
@agliga Given that a badge only applies to menus, I wonder if this change should be made in menu & menu-button, rather than the dropdown mixin? Not a huge deal either way, if you think it's better for consistency like this.